environments/demo/demo-platform-hmcts-net.yml

```diff
@@ -42,17 +42,29 @@ cname:
     ttl: 300
     record: "s2.domainkey.u24999805.wl056.sendgrid.net"
   - name: vh-video-web
-    record: sdshmcts-demo.azurefd.net
+    record: sdshmcts-stg.azurefd.net
```

Comment:
stg?

Reply:
copy and paste issue. I have corrected them now